apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/KafkaConsumerInterceptor.java

```diff
@@ -44,6 +48,17 @@
 
     public static final String OPERATE_NAME_PREFIX = "Kafka/";
     public static final String CONSUMER_OPERATE_NAME = "/Consumer/";
+    private static final Gson GSON = new Gson();
+
+    private static boolean TRACE_MSG_OPEN;
+
+    public KafkaConsumerInterceptor() {
+        try {
+            Class.forName("org.apache.skywalking.apm.toolkit.trace.msg.TraceMsg");
```

Comment:
This is totally wrong.

Reply:
Thank you for your guidance.
The original target here, The KafkaConsumerInterceptor can automatically stores trace information into the message that are implemented the TraceMsg interface.

I will optimize this code;  The pseudo-code used is as follows:
1.  String trace=TraceMsgContext.getCurretTrace();    //provided to obtain trace information
2.  msg.setTraceData(trace);  // customized trace data  in message body
3. TraceMsgContext.ConsumerMsgSucceed(msg.getTraceData()) //trace the real result of kafka message processing

In addition, for the scenarios supported by this feature, whether it is valuable~? 
I submit a discussion  later;

Thank you again for your guidance; and Thank SkyWalking for bringing a lot of value to the project;

Reply:
Please submit a discussion to sync the case you faced and try to resolve.
Then we could move on tech details, finally as a pull request.
Right now, I am confused with this PR.